flambe/metric/metric.py

```diff
@@ -12,24 +12,11 @@ class Metric(Component):
     examples and provide as output a processd list of the same size.
 
     """
-    @abstractmethod
-    def compute(self, pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
-        """Computes the metric over the given prediction and target.
-
-        Parameters
-        ----------
-        pred: torch.Tensor
-            The model predictions
 
-        target: torch.Tensor
-            The ground truth targets
-
-        Returns
-        -------
-        torch.Tensor
-            The computed metric
-
-        """
+    @typing.no_type_check
+    @abstractmethod
+    def compute(self, *args, **kwargs):
```

Comment:
This is a strong change. Can't we relax the typing but not remove them completely? For example, are we always returning `Tensor` right?

Reply:
The better question is: why restrict the type?